drivers/usb/udc/Kconfig.stm32

```diff
@@ -6,6 +6,7 @@ config UDC_STM32
 	depends on DT_HAS_ST_STM32_OTGFS_ENABLED \
 		|| DT_HAS_ST_STM32_OTGHS_ENABLED \
 		|| DT_HAS_ST_STM32_USB_ENABLED
+	depends on !USB_DC_STM32
```

Comment:
It's also a bit specific to this driver though - at least theoretically it should be possible with multi-instance compatible controllers to use one with the new api, and one with the old. But the two stm32 drivers specifically cannot coexist because of the Hal callback definitions.

Reply:
ok, Thanks. Fine for me then